core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -307,6 +309,103 @@ class MultiCloudWorkspaceService(override val ctx: RawlsRequestContext,
       } yield profileModel
     }
 
+  def cloneMultiCloudWorkspaceAsync(
```

Comment:
This is all just straight up duplicated, for now; up to the call to `cloneAzureWorkspaceAsync` (instead of `cloneAzureWorkspace`).
When we're ready to kill the old path we can just change the call in `cloneMultiCloudWorkspace`, and remove this altogether.